.gitignore

```diff
@@ -0,0 +1,266 @@
+# Created by https://www.toptal.com/developers/gitignore/api/django,python
+# Edit at https://www.toptal.com/developers/gitignore?templates=django,python
+
+### Django ###
+*.log
+*.pot
+*.pyc
```

Comment:
The line '*.pyc' ensures that compiled Python files are ignored, which aligns with the checklist requirement: 'Make sure you don't push .pyc files.'